app/src/main/java/org/mozilla/focus/telemetry/TelemetryWrapper.kt

```diff
@@ -129,6 +129,8 @@ object TelemetryWrapper {
         val RELOAD = "refresh"
         val BACK = "back"
         val FORWARD = "forward"
+        val HOME = "home"
```

Comment:
Do you listen for the physical menu button press?

Reply:
Not at present: I filed #354 for the menu.